### PR TITLE
Change Price from Integer to Double (allow up to 2dp)

### PR DESCRIPTION
**Pull request by kailinteoo**

### CODE REVIEW

File: src/main/java/seedu/address/model/order/Price.java [hunk 1]
```diff
@@ -11,7 +14,7 @@ public class Price {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Price must be a positive number";
```
Comment (Jaspertzx) [inline src/main/java/seedu/address/model/order/Price.java:16]: Could you change this constraint to be more specific? such as "Price must be a positive number and can only include up to two decimal places."

File: src/test/java/seedu/address/model/order/PriceTest.java [hunk 2]
```diff
@@ -37,18 +37,18 @@ void constructor_invalidPriceZero_throwsIllegalArgumentException() {
...
         assertFalse(Price.isValidPrice("-1"));
 
         assertThrows(NumberFormatException.class, () -> Price.isValidPrice(" "));
         assertFalse(Price.isValidPrice("0"));
-        assertThrows(NumberFormatException.class, () -> Price.isValidPrice("1.5"));
-        assertThrows(NumberFormatException.class, () -> Price.isValidPrice("1.0"));
 
         assertTrue(Price.isValidPrice("1"));
         assertTrue(Price.isValidPrice("100"));
+        assertTrue(Price.isValidPrice("1.5"));
```
Comment (Jaspertzx) [inline src/test/java/seedu/address/model/order/PriceTest.java:50]: Could you add some test cases for 3 decimal places?

**Review by Jaspertzx — CHANGES_REQUESTED** [review]
Almost good! Just some minor adjustments needed before I approve

File: src/test/java/seedu/address/model/order/PriceTest.java (hunk 2 above)
Comment (kailinteoo) [reply src/test/java/seedu/address/model/order/PriceTest.java:50]: you are right, I will include that. Thank you for mentioning

Comment (kailinteoo) [reply src/test/java/seedu/address/model/order/PriceTest.java:50]: Thank you for pointing out. I will fix it and push again.

**Review by Jaspertzx — APPROVED** [review]

**Pull request merged**